src/components/trello/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './style.scss';
+import { Card } from 'antd';
+
+export type titleType = {
+  id: number;
+  title: string;
+};
+
+export type dataType = {
+  cards: titleType[];
+  addCardTitle: () => React.ReactNode;
+};
+const Trello: React.FC<dataType> = ({ cards, addCardTitle }) => {
+  return (
+    <div className="main">
+      <div className="container">
+        <div className="main__grid">
+          <Card className="main__card" title="Todo">
+            {cards.map((elemen) => {
```

Comment:
если это cards, следовательно должно быть card внутри map
```suggestion
            {cards.map((card) => {
```